.prettierrc.json

```diff
@@ -1,4 +1,10 @@
 {
   "singleQuote": false,
-  "printWidth": 90
+  "printWidth": 90,
+  "semi": true,
```

Comment:
`"semi": false`, I don't like trailing semicolons and they are not a big issue with eslint and Typescript.